app/src/main/java/com/firebase/uidemo/database/ChatActivity.java

```diff
@@ -70,7 +71,10 @@ protected void onCreate(Bundle savedInstanceState) {
         mSendButton = (Button) findViewById(R.id.sendButton);
         mMessageEdit = (EditText) findViewById(R.id.messageEdit);
 
-        mChatRef = FirebaseDatabase.getInstance().getReference().child("chats");
+        mEmptyListView = findViewById(R.id.emptyTextView);
+
+        mRef = FirebaseDatabase.getInstance().getReference();
+        mChatRef = mRef.child("chats");
```

Comment:
@puf looks like the merge didn't work (`mChatRef` got split into a nonexistent `mRef` for some reason).